ote/src/clj/ote/services/reports.sql

```diff
@@ -3,10 +3,12 @@
 -- name: fetch-operators-no-services
 SELECT op.name, op.id, op.phone, COALESCE(NULLIF(op.email, ''), u.email) AS "email"
   FROM "transport-operator" op
-         JOIN "user" u ON u.name = (SELECT author
-                                      FROM "revision" r
-                                             JOIN "group" g ON op."ckan-group-id" = g.id
-                                     WHERE r.id = g."revision_id"
+         JOIN "user" u ON u.name = (SELECT u.name
```

Comment:
Ok, but for clarity it would be good to not to use "`u`" twice as alias but for different tables.